mlir/lib/Dialect/SparseTensor/IR/SparseTensorDialect.cpp

```diff
@@ -1454,6 +1486,14 @@ LogicalResult ToPositionsOp::verify() {
   return success();
 }
 
+LogicalResult
+ToPositionsOp::inferReturnTypes(MLIRContext *ctx, std::optional<Location> loc,
```

Comment:
Will there be multiple return types? Since inferSparseBufferType is singular, should inferReturnType be singular too?